modules_basic/avatar/image.js

```diff
@@ -74,6 +76,24 @@ exports.create = function (api) {
       )
     },
 
+    avatar_image_src: function (author) {
+      var src = ready && avatars[author] ? api.blob_url(avatars[author].image) : genSrc(author)
+
```

Comment:
no this won't work (if I understand what you are trying to do...) because you are gonna assign a image's src property to this right? remember the variable `src` is just a reference so when you do `image.src = src` then later set `src = src2` `image.src` will still have the old value.